nextcord/guild.py

```diff
@@ -3204,8 +3208,8 @@ async def create_scheduled_event(
         privacy_level: ScheduledEventPrivacyLevel = ScheduledEventPrivacyLevel.guild_only,
         end_time: datetime.datetime = MISSING,
         description: str = MISSING,
-        image: bytes = MISSING,
-        reason: Optional[str] = None,
+        image: Union[bytes, Asset, Attachment] = None,
```

Comment:
Denote Optional if it accepts None

```suggestion
        image: Optional[Union[bytes, Asset, Attachment]] = None,
```